src/clj/game/cards-identities.clj

```diff
@@ -370,7 +370,8 @@
                               :req (req (has-subtype? target "Virus"))}}}
 
    "Pālanā Foods: Sustainable Growth"
-   {:events {:runner-draw {:msg "gain 1 [Credits]"
+   {:events {:runner-draw {:req (req (not= 0 (:turn @state)))
```

Comment:
I don't think putting an exception for mulligan turn is the right approach. It'd probably be more desirable to make mulligan draws not trigger any effects, since taking new cards due to mulligan isn't really a draw in normally used game terms.